Content.Shared/Movement/Components/JumpComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Content.Shared.Actions;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+
+namespace Content.Shared.Movement.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class JumpComponent : Component
+{
+    /// <summary>
+    /// How powerful the jump will be when activated
+    /// </summary>
+    [DataField("jumpPower"), ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+    public float JumpPower = 5f;
+
+
+    // Logical variables
+    [DataField]
+    public EntProtoId Action = "ActionGravityJump";
+
+    [DataField]
+    public EntityUid? ActionEntity;
+
+    [DataField]
+    public bool? IsClothing;
```

Comment:
> why make a bool nullable?

Same thing, it's so that the ability can be applied to both a garment with a component and a character with a component, so this variable is needed to check if it's a clothing or not

Reply:
But why use true/false/null if true/false is all that is needed?

Reply:
> But why use true/false/null if true/false is all that is needed?

Hmm...sounds logical. I'll change it then